docs/5.x/changelog.md

```diff
@@ -8,8 +8,8 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 | Release       | LTS | Release Date         | Supported Until      | Kubernetes Version | Teleport Version |
 | --------------|-----| -------------------- | -------------------- | ------------------ |------------------|
 | 5.5.0-alpha.6 | No  | January, 15th 2019   | -                    | 1.12.3             | 3.0.1            |
```

Comment:
Btw, Kubernetes version in master is also 1.13.0 so we should update it in this table.